runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomConfig.java

```diff
@@ -42,11 +42,22 @@ default boolean innerLimit()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "outerLimit",
+		name = "Expand outer zoom limit",
+		description = "Configures whether or not the outer zoom limit is increased",
+		position = 2
+	)
+	default int outerLimit()
```

Comment:
I'm assuming this is not meant to be an int? Either that, or the description is woefully unhelpful.